packages/console/src/hooks/use-action-translation.ts

```diff
@@ -0,0 +1,20 @@
+import { type AdminConsoleKey } from '@logto/phrases';
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+const decapitalize = (value: string) => value.charAt(0).toLowerCase() + value.slice(1);
```

Comment:
Use `toLocaleLowerCase()` instead?

Reply:
it has a different meaning comparing to decapitalize?